pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -211,8 +211,10 @@ func getDaemonPoshScriptContent() []byte {
 	if err != nil {
 		return []byte{}
 	}
+	drive := filepath.VolumeName(binPath)
+	binPathWithSmallCaseDriveLetter := strings.Replace(binPath, drive, strings.ToLower(drive), 1)
 	daemonCmdArgs := `daemon --log-level debug`
-	return []byte(fmt.Sprintf(daemonPoshScriptTemplate, binPath, daemonCmdArgs))
+	return []byte(fmt.Sprintf(daemonPoshScriptTemplate, binPathWithSmallCaseDriveLetter, daemonCmdArgs))
```

Comment:
I don't think we should assume other program always going to use the small case driver letter so, I think it is other program responsibility to first check the crc executable because after crc is installed we add that path so other program should able to detect the binary.